tools-local/unix/invoke-extension.sh

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env bash
+
+# Locates and invokes the given extension script, which does a search or acquire, with the specified arguments. 
+
+usage()
+{
+    echo "Usage: $0 ScriptName ToolName -StrictToolVersionMatch --OverrideScriptsFolderPath <path>"
```

Comment:
I haven't seen one, but this comment is about the args, not the usage doc. My example is meant to show that `--` is "end of `invoke-extension arguments`" and extra arguments follow after that.

If *all* `invoke-extension` args can be positional, though, `--` is unnecessary, and instead "all args after 3" are extra arguments. See https://github.com/ravimeda/corefx/pull/7#discussion_r109716811